CHANGELOG.md

```diff
@@ -1,3 +1,63 @@
+# 84.3.0
+
+This minor release adds new theming extensions for Listsz and minior bug fixes and documentation updates to multiple components.
+
+## New features
+
+# Dynamic type support
```

Comment:
We already had support for Dynamic Type, but the difference is now we're supporting Dynamic Type using MDCFontScaler.  I think we should be careful in this wording so it doesn't sound like we didn't have any support previously.

Reply:
Done